frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -470,7 +509,7 @@ export const createOrUpdateResources = async (
   const defaultAnnotations = getGitAnnotations(repository, ref);
 
   if (pipeline.enabled && pipeline.template && !dryRun) {
-    responses.push(await createPipelineForImportFlow(formData));
+    responses.push(await managePipelineResources(formData));
```

Comment:
Technically this is a wee bit of a functionality change. `pipeline.template` is not a prereq for the secret stuff in 4.7+... but I think it's okay since it's a slimmed down functionality for 4.6.